CYD-Klipper/src/ui/ip_setup.cpp

```diff
@@ -33,6 +33,21 @@ static const lv_btnmatrix_ctrl_t kb_ctrl[] = {
     LV_KEYBOARD_CTRL_BTN_FLAGS | 6, 4, 4, 4, 4, 4, 4, 4, 4, 4, LV_KEYBOARD_CTRL_BTN_FLAGS | 6
 };
 
+static const char * hex_numpad_map[] = {
+    "1", "2", "3", "f", LV_SYMBOL_BACKSPACE, "\n",
+    "4", "5", "6", "e", LV_SYMBOL_OK, "\n",
+    "7", "8", "9", "d", LV_SYMBOL_LEFT, "\n",
+    "0", "a", "b", "c", LV_SYMBOL_RIGHT, "\n",
```

Comment:
Is this last \n needed?

Reply:
it is not... that explains the empty keyboad row at the bottom. Much better now.

![PXL_20240428_152707583](https://github.com/suchmememanyskill/CYD-Klipper/assets/5213469/9e8ab576-7311-4a22-8b63-afe3e928fee0)